flink-staging/flink-streaming/flink-streaming-core/src/test/java/org/apache/flink/streaming/api/IterateTest.java

```diff
@@ -87,7 +90,10 @@ public void test() throws Exception {
 
 		env.execute();
 
-		assertTrue(iterated);
+		for(boolean iter : iterated)
```

Comment:
we usually have the left curly brace in the same line (I know we don't have any official rule enforcing this ;) )